tests/geometry/volumes/distance/testReducedMedialAxis.cpp

```diff
@@ -52,26 +53,26 @@ bool testReducedMedialAxis()
 {
   unsigned int nbok = 0;
   unsigned int nb = 0;
-  
+
   trace.beginBlock ( "Testing PowerMap2D ..." );
 
   Z2i::Domain domain(Z2i::Point(0,0),Z2i::Point(10,10));
   Z2i::Domain domainLarge(Z2i::Point(0,0),Z2i::Point(10,10));
 
   DigitalSetBySTLSet<Z2i::Domain > set(domain);
-  set.insertNew(Z2i::Point(3,3)); 
-  //set.insertNew(Z2i::Point(3,7)); 
+  set.insertNew(Z2i::Point(3,3));
+  //set.insertNew(Z2i::Point(3,7));
   set.insertNew(Z2i::Point(7,7));
-  DigitalSetDomain<DigitalSetBySTLSet<Z2i::Domain > > setDomain(set); 
-  
-  typedef ImageContainerBySTLMap<DigitalSetDomain<DigitalSetBySTLSet<Z2i::Domain > > , DGtal::int64_t> Image;
-  Image image(setDomain);
-  
+
+  using SetDomain = DigitalSetDomain< DigitalSetBySTLSet<Z2i::Domain > >;
+  using Image = ImageContainerBySTLMap< SetDomain , DGtal::int64_t>;
+  Image image( CountedPtr<const SetDomain>( new SetDomain( set ) ) );
```

Comment:
If you wish here, you can just pass "new SetDomain( set )" which will be acquired by the image because of the Clone<Domain>.